mapnik_c_api.cpp

```diff
@@ -257,6 +280,105 @@ const char *mapnik_image_last_error(mapnik_image_t *i) {
     return NULL;
 }
 
+void process_vector_data(mapnik::agg_renderer<mapnik_rgba_image> & ren, mapnik_map_t * m, mapnik_vector_data_t * data){
```

Comment:
This is roughly ported from [node-mapnik](https://github.com/mapnik/node-mapnik/blob/ea012648beb476aafc747732e955027c99212c4c/src/mapnik_vector_tile.cpp#L3796)